yt_dlp/extractor/lsm.py

```diff
@@ -0,0 +1,314 @@
+                'duration': 3222,
+            },
+        }, {
+            'md5': '5d5e191e718b7644e5118b7b4e093a6d',
+            'info_dict': {
+                'id': 'v303104',
+                'ext': 'mp4',
+                'thumbnail': 'https://pic.latvijasradio.lv/public/assets/media/c/5/gallery_a83ad2c2.jpg',
+                'title': 'Krustpunktā Lielā intervija: Valsts prezidents Egils Levits - Video Version',
+                'duration': 3222,
+            },
+        }],
+    }, {
+        'url': 'https://lr1.lsm.lv/lv/embed/?id=183522&show=0&theme=white&size=16x9',
+        'only_matching': True,
+    }, {
+        'url': 'https://lr2.lsm.lv/lv/embed/?id=182126&show=0&theme=white&size=16x9',
+        'only_matching': True,
+    }, {
+        'url': 'https://klasika.lsm.lv/lv/embed/?id=110806&show=0&theme=white&size=16x9',
+        'only_matching': True,
+    }, {
+        'url': 'https://lr4.lsm.lv/lv/embed/?id=184282&show=0&theme=white&size=16x9',
+        'only_matching': True,
+    }, {
+        'url': 'https://pieci.lv/lv/embed/?id=168896&show=0&theme=white&size=16x9',
+        'only_matching': True,
+    }, {
+        'url': 'https://naba.lsm.lv/lv/embed/?id=182901&show=0&theme=white&size=16x9',
+        'only_matching': True,
+    }, {
+        'url': 'https://radioteatris.lsm.lv/lv/embed/?id=176439&show=0&theme=white&size=16x9',
+        'only_matching': True,
+    }, {
+        'url': 'https://lr1.lsm.lv/lv/pleijeris/?embed=0&id=48205&time=00%3A00&idx=0',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        query = parse_qs(url)
+        video_id = traverse_obj(query, ('show', 0, {int_or_none}))
+
+        if not video_id:
+            video_id = traverse_obj(query, ('id', 0, {int_or_none}))
+
+        webpage = self._download_webpage(url, video_id)
+
+        player_data, media_data = self._html_search_regex(
+            r'LR\.audio\.Player\s*\([^{]*(?P<player>\{.*?\}),(?P<media>\{.*\})\);',
+            webpage, 'player json', group=('player', 'media'))
+
+        player_json = self._parse_json(player_data, video_id, js_to_json)
+        media_json = self._parse_json(media_data, video_id, js_to_json)
+
+        entries = []
+        for item in traverse_obj(media_json, (('audio', 'video'), ...)):
+            formats = []
+            for source_url in traverse_obj(item, ('sources', ..., 'file', {url_or_none})):
+                if source_url.endswith('.m3u8'):
+                    formats.extend(self._extract_m3u8_formats(source_url, video_id))
+                else:
+                    formats.append({'url': source_url})
+
+            id = item.get('id')
+            title = item.get('title')
+            if id and id.startswith('v') and not title:
+                title = traverse_obj(
+                    media_json,
+                    ('audio', lambda _, v: v['id'][1:] == id[1:], 'title', {lambda x: x and f'{x} - Video Version'}),
+                    get_all=False)
```

Comment:
I would format like so, but not sure:
```suggestion
                title = traverse_obj(
                    media_json, ('audio', lambda _, v: v['id'][1:] == id[1:], 'title',
                                 {lambda x: x and f'{x} - Video Version'}), get_all=False)
```